custom/icds_reports/management/commands/build_disha_dumps.py

```diff
@@ -25,6 +25,13 @@ def valid_date(s):
             type=valid_date,
             help="Month of DISHA dump - format = YYYY-MM-DD",
         )
+        parser.add_argument(
+            '--force-rebuild',
```

Comment:
how about `--all-states`?

Reply:
Even without this option, it will build for all states (but skips if a state already has a build). This literally makes a force rebuild of even states that have the data. So I think this is a suitable title here

Reply:
I will update the help text if that confused you.

Reply:
i think thats fine. thanks